charts/seed-bootstrap/dashboards/fluent-bit-dashboard.json

```
@@ -0,0 +1,687 @@
+{
+  "annotations": {
+    "list": []
+  },
+  "description": "Fluent-bit metrics",
+  "editable": true,
+  "gnetId": 7752,
+  "graphTooltip": 1,
+  "links": [],
+  "panels": [
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "prometheus",
+      "description": "Input bytes processing rate of all fluent-bits.",
+      "fieldConfig": {
+        "defaults": {
+          "custom": {}
+        },
+        "overrides": []
+      },
+      "fill": 5,
```

```suggestion
      "fill": 0,
```
Fill 0 is easier to read IMO. I will leave it up to you if you want to change it or not. If you do change it please change it for all panels.

---

Done